zb-db/src/main/java/io/zeebe/db/impl/rocksdb/transaction/ZeebeTransactionDb.java

```diff
@@ -136,6 +143,17 @@ protected ZeebeTransactionDb(
     this.handelToEnumMap = handelToEnumMap;
     this.closables = closables;
     this.columnFamilyNamesClass = columnFamilyNamesClass;
+
+    prefixReadOptions = new ReadOptions().setPrefixSameAsStart(true).setTotalOrderSeek(false);
+    closables.add(prefixReadOptions);
+    simpleReadOptions = new ReadOptions();
+    closables.add(simpleReadOptions);
+    final WriteOptions simpleWriteOptions = new WriteOptions();
```

Comment:
Again please call it `defaultWriteOptions` as it is just the default values and nothing simple